README.md

```diff
@@ -19,10 +19,10 @@ KubeVirt Ansible consists of a set of Ansible playbooks that deploy fully functi
 ## Usage
 
 ### Deploy
-To deploy KubeVirt on an existing OpenShift cluster run the command below. For more information on clusters and other deployment scenarious see [playbooks instructions](./playbooks/README.md).
+To deploy KubeVirt on an existing OpenShift cluster run the command below. For more information on clusters and other deployment scenarios see [playbooks instructions](./playbooks/README.md).
 
 ```
-ansible-playbook -i localhost playbooks/kubevirt.yml -e@vars/all.yml
+ansible-playbook -i localhost playbooks/kubevirt.yml -e@vars/all.yml --skip-tags=never
```

Comment:
this looks like it skips [this code](https://github.com/kubevirt/kubevirt-ansible/blob/25d96207047cd4ba046f36738584681031817eec/roles/kubevirt/tasks/provision.yml#L57).  It would be better to fix the demo-content block or add a when statement that's defaulted to off around the demo-content block instead of having to add `--skip-tags=never`.

Reply:
@booxter thoughts?

Reply:
@rthallisey I have no immediate plans to fix demo manifests. If I should use when statement defaulted to off, what do you think should be the statement itself? Should I introduce some new fact like 'deploy_demo' that would be off by default?

Reply:
`deploy_demo: false` works for me.